src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerUdpBatcher.cs

```diff
@@ -73,8 +73,15 @@ public JaegerUdpBatcher(JaegerExporterOptions options, TTransport clientTranspor
 
             this.maxFlushIntervalTimer.Elapsed += async (sender, args) =>
             {
-                await this.FlushAsyncInternal(false, CancellationToken.None).ConfigureAwait(false);
-            };
+                try

```

Comment:
I think it would be better to move this try/catch block inside of the `FlushAsyncInternal` function as there are a few of places that can call it, eg event timer (here), AppendAsync, FlushAsync and CloseAsync.

Reply:
Yes, but problem itself not in exception inside FlushAsyncInternal, exceptions from timer tick event handler not handling in any other places. If someone add here more code we get problem again. I can add catch with exception log in other try\finally blocks

Reply:
Changed, take a look